### DISCUSSION TITLE
Can Dax Studio Analyze a file that is NOT saved in metadata format?

### DISCUSSION THREAD

**Discussion by Siouxdr** [DaxStudio/DaxStudio#640] [Q&A]
I have been working in a Power BI Report Server environment and we have not enabled metadata format in our current desktop version-we did this because the metadata format in a previous desktop for RS version caused issues with refreshing of datasets.  In the file options menu for report server metadata is considered a preview feature-and it is shut off.  When I try to analyze a file in PBI with Dax studio, it appears to be failing to make a connection-is this because our desktop version is not saving the data in metadata format?

**Comment by dgosbell** [comment]
No the internal metadata format does not matter you should be able to connect to these files without any problem. 

What do you mean by "it appears to be failing to make a connection"? Are you getting an error message of some sort? If so it would help to know what that error message is.

**Reply by Siouxdr** [reply]
Yes ,it just seems to hang-the model is very small-see enclosed pic
![Dax](https://user-images.githubusercontent.com/82285852/120494143-8e48be80-c389-11eb-9030-9dac03da1796.PNG)

**Reply by Siouxdr** [reply]
It just says establshing connection but does not appear to process the data in the model like it used to in long ago versions of Dax Studio

**Reply by dgosbell** [reply]
That screen shot looks like everything is fine. There is no error in the output pane and I can see the model metadata on the left and and the ribbon buttons are enabled as they would be once a successful connection is established. I suppose I could add a "Connection successful" message in the output, but we've never had that in the past. 

It looks like everything is connected and ready for you to start running queries. 

Are you saying you the application is frozen and you can't click on any buttons or type in the query pane or open the query builder?

I'm not sure what you are expecting when you say "but does not appear to process the data in the model like it used to" we've never automatically done anything with the data when you first connect. Maybe you could have a look at some of the [DAX Studio videos](https://www.sqlbi.com/p/dax-tools-video-course/) at sqlbi.com and see if they help

**Reply by Siouxdr** [reply]
When I export the VPAX file, and open analysis, it gets stuck in a perpetual reading data loop and does not ever finish.  The screenshot below is from a file with two data tables (Calendar and Fact), with 950 rows of data.  I have run these analyses before prior on Dax Studio and have not had these issues previously-please see screenshot below-I think this version of Dax Studio is having some compatability problems with Power BI Report Server (v Jan 2020 optimized
 for desktop)

**Reply by Siouxdr** [reply]
Screen shot of vertipaq hanging

**Reply by Siouxdr** [reply]
![image](https://user-images.githubusercontent.com/82285852/121436103-f2e9b780-c94d-11eb-8d8c-f39b3499a995.png)

**Reply by marcosqlbi** [reply]
It seems you are using an old version of VertiPaq Analyzer - can you try the latest version?
https://www.sqlbi.com/tools/vertipaq-analyzer/

**Reply by dgosbell** [reply]
Can you import the vpax file back into DAX Studio? If you can that would confirm that the problem is most likely in the Excel version.

**Comment by Siouxdr** [comment]
Good morning

Same issue-I have downloaded the most current version of Vertipaq-same issue-just will not load the Vpax file correctly.  It just hangs and does not analyze the file.  This is an extremely small file, so I do not think that is contributing to this.

I can attach the vpax file if that will help-I think there may be some kind of issue with reading the data out of a PBIX file that is based on PBI Report Server for Desktop-we are currently on January 2021 version

Thank very much,

**Comment by Siouxdr** [comment]
Hi Darren

Yes, I can import that back into Dax Studio

***@***.***D75DC9.C3667380]Curiously, when I do so, I can see the data I normally would see in the results pane below now where I could not when I ran Dax studio to get the VPAX file generated.

***@***.***D75DC9.C3667380]

But I still cannot run the VPAX file properly in Vertipaq-with the most current version-it still is hanging and not reading the data in properly.

Thanks,

**Reply by dgosbell** [reply]
> Curiously, when I do so, I can see the data I normally would see in the results pane below now where I could not when I ran Dax studio to get the VPAX file generated.

This is expected, when you click export we don't also load the data into the UI I just send it directly to the vpax file.

So it sounds like this issue is isolated to the excel version, if you can share a file that exhibits this behaviour that would help. It's unlikely that using the RS version of Desktop is the problem, but it's hard to say what is happening without being able to observe this behaviour and examine the file more closely.
